giskard_cicd/pipeline/runner.py

```diff
@@ -33,8 +33,8 @@ def run(self, loader_id, **kwargs):
         if scan_config_path is not None:
             with open(scan_config_path) as yaml_f:
                 scan_config = yaml.load(yaml_f, Loader=yaml.Loader)
-            params = dict(scan_config.get("configuration", None))
-            detectors = list(scan_config.get("detectors", None))
+            params = dict(scan_config.get("configuration", {}))
+            detectors = list(scan_config.get("detectors"))
```

Comment:
This could be `None` to run all the detectors:
https://github.com/Giskard-AI/giskard/blob/0c58a9bb29c9b91ac75e774eb92ad54677f3be31/giskard/scanner/scanner.py#L301

Reply:
None would cause list() to throw error. I fixed these changes with better implementation.